config.yaml

```diff
@@ -105,6 +105,14 @@ options:
   contracts.password:
     description: Password to authenticate with backend contracts service.
     type: string
+  contracts.ca:
```

Comment:
Why base64 btw? The cert is not binary data

Reply:
other charms do the same..